### DISCUSSION TITLE
How to Assign S Parameter to Specific Component in 3D Layout with PyAEDT？

### DISCUSSION THREAD

**Discussion by linmingchih** [ansys/pyaedt#4363] [Q&A]
I can't find the API that can assign S parameter on component, please help. Thanks.

![image](https://github.com/ansys/pyaedt/assets/13580731/b5e54c41-482d-4d7a-bbaf-6efba53fd5ac)

**Comment by Samuelopez-ansys** [comment]
Hi @linmingchih ,

This method is not yet implemented. Please submit an issue if you need it, but in the meantime, you can record the script and add it to your workflow with PyAEDT.

You can try to contribute and add it inside PyAEDT :)
